### PR TITLE
Validation fix to allow for string numbers

### PR DESCRIPTION
**Pull request by snurby7**
# Pull Requests

Please accompany all pull requests with the following (where appropriate):
- [x] unit tests (we use [Jasmine 1.3](http://jasmine.github.io/1.3/introduction.html))
- [x] JSFiddle (plunker [link ](https://plnkr.co/edit/TRhGSSQu9yCluH7xDHNC?p=preview) [toggle the two files, new is update]
- [x] Fix for #630 
- [ ] documentation updates to README file
- [ ] examples within [/tpl/index.tpl](https://github.com/seiyria/bootstrap-slider/blob/master/tpl/index.tpl) (for new options being added)
- [x] Passes CI-server checks (runs automated tests, JS source-code linting, etc..). To run these on your local machine, type `grunt test` in your Terminal within the bootstrap-slider repository directory

### CODE REVIEW

**Comment by seiyria** [comment]
-1. IMO validate your data before passing it in.

**Comment by snurby7** [comment]
@seiyria fair enough, (I do agree that it should be cleaned prior). You guys seemed to be open to the PR in #630  so I just wrapped it all up.

**Comment by seiyria** [comment]
Sorry, some days I'm on autopilot. 

It's ultimately up to @rovolution but I think it's reasonable to require that the slider only operate on numbers.

File: test/specs/PublicMethodsSpec.js [hunk 1]
```diff
@@ -246,6 +246,14 @@ describe("Public Method Tests", function() {
         expect(sliderValue).toBe(valueToSet);
       });
 
+      it("properly sets the value of the slider when given a string value", function(){
```
Comment (rovolution) [inline test/specs/PublicMethodsSpec.js]: could you also add a test for when you pass a string value that cannot be properly converted into a number?

For example: 

```js
testSlider.slider('setValue', "rolo");
```

**Comment by snurby7** [comment]
@rovolution added an additional test and updated the readme. I feel as though the test is a little redundant as down in the file a little ways before the range slider tests there's a test "when an invalid value type is passed in" that passes in "a" as the value and throws and error, I can remove it if you feel the same way.

**Comment by rovolution** [comment]
@snurby7 oops just saw that test! Thanks for pointing that out! Yea for sure go ahead and delete that test then since it would be redundant

**Pull request merged**